File: jvm2/src/Model/Neutral/Buildings/Factory.kt
package Model.Neutral.Buildings

import Model.Abstract.RealTile

abstract class Factory() : Building() {
    override var minCost = 200
    override var gasCost = 100
    override var nrgCost = 0
    override var commands = mutableListOf<String>("Create a Builder (50, 20, 0)", "Create a Worker (20, 10, 0)", "Abort")
    override var health = 100
}

File: dotnet/src/Model/Neutral/Buildings/Factory.cs
using System;
using System.Collections.Generic;

namespace PlanetCS1 {
    public abstract class Factory : Building {
        public Factory() {
            this.minCost = 200;
            this.gasCost = 100;
            this.nrgCost = 0;
            this.commands = new List<String>(){"Create a Builder (50, 20, 0)", "Create a Worker (20, 10, 0)", "Abort"};
            this.health = 100;
        }
    }
}